dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/ProjectPage.java

```diff
@@ -65,36 +69,50 @@ public ProjectPage create(String project) {
         createProjectForm().buttonSubmit().click();
 
         new WebDriverWait(driver(), 10)
-            .until(ExpectedConditions.textToBePresentInElementLocated(By.className("project-name"), project));
+                .until(ExpectedConditions.textToBePresentInElementLocated(By.className("project-name"), project));
+
+        return this;
+    }
+
+    public ProjectPage create(String project, String describe) {
```

Comment:
Project description is not that important and we don't want to cover every aspect of the ui so I think we don't need the test of creating project with description. It costs resources without bringing too many benefits.

Reply:
> Project description is not that important and we don't want to cover every aspect of the ui so I think we don't need the test of creating project with description. It costs resources without bringing too many benefits.

E2E task has this item, so is there any E2E task that can be done, please give it?